cmd/mlbviolations/main.go

```diff
@@ -155,3 +198,14 @@ func main() {
 		}
 	}
 }
+
+func getViolationCount(db *sql.DB) int {
+	var seasonTotal int
+	err := db.QueryRow(fmt.Sprintf("SELECT COUNT(*) FROM player_stats")).Scan(&seasonTotal)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return seasonTotal
```

Comment:
So I think in a more idiomatic go world, you would return the err and nil here and log fatal in the calling code.

Then all the exit points are clear from `main`